docs/dev/table/connectors/print.zh.md

```diff
@@ -104,40 +104,40 @@ LIKE source_table (EXCLUDING ALL)
 </div>
 </div>
 
-Connector Options
+连接器选项
 ----------------
 
 <table class="table table-bordered">
     <thead>
       <tr>
-        <th class="text-left" style="width: 25%">Option</th>
-        <th class="text-center" style="width: 8%">Required</th>
-        <th class="text-center" style="width: 7%">Default</th>
-        <th class="text-center" style="width: 10%">Type</th>
-        <th class="text-center" style="width: 50%">Description</th>
+        <th class="text-left" style="width: 25%">选项名字</th>
+        <th class="text-center" style="width: 10%">是否可选</th>
+        <th class="text-center" style="width: 10%">默认参数</th>
+        <th class="text-center" style="width: 10%">数据类型</th>
+        <th class="text-center" style="width: 45%">描述</th>
       </tr>
     </thead>
     <tbody>
     <tr>
       <td><h5>connector</h5></td>
-      <td>required</td>
+      <td>必选</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
-      <td>Specify what connector to use, here should be 'print'.</td>
+      <td>指定要使用的连接器，此处应为 'print'</td>
     </tr>
     <tr>
       <td><h5>print-identifier</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
-      <td>Message that identify print and is prefixed to the output of the value.</td>
+      <td>配置一个标识符作为输出数据的前缀。</td>
     </tr>
     <tr>
       <td><h5>standard-error</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;">false</td>
       <td>Boolean</td>
-      <td>True, if the format should print to standard error instead of standard out.</td>
+      <td>如果格式应打印为标准错误而不是标准输出，则为 True 。</td>
```

Comment:
Please keep 'format' untranslated. I think format is an Object here rather than style.